### ISSUE TITLE
Installation Error NB 8.2

### ISSUE THREAD

**Issue by joaopauloandrade** [jeddict/jeddict#254]
##### **Overview of the issue**

I'm trying to install Jeddict in the current scenario:
NetBeans:
Product Version: NetBeans IDE 8.2 (Build 201609300101)
Updates: NetBeans IDE is updated to version NetBeans 8.2 Patch 2
Java: 1.8.0_121; Java HotSpot(TM) 64-Bit Server VM 25.121-b13
Runtime: Java(TM) SE Runtime Environment 1.8.0_121-b13
System: Mac OS X version 10.13.1 running on x86_64; UTF-8; en_US (nb)
User directory: /Users/joaopaulo/Library/Application Support/NetBeans/8.2
Cache directory: /Users/joaopaulo/Library/Caches/NetBeans/8.2

Jeddict 
1493140630_Jeddict_v4.2.1_NBv8.2.zip

23 plugins where selected. When i click on install, get the error above:

"Some plugins require plugin Docker API to be installed
The plugin Docker API is requested in implementation version 201609300101
The following plugin is affected
      Docker

Some plugins require plugin Docker UI to be installed
The plugin Docker UI is requested in implementation version 201609300101
The following plugin is affected
      Docker

**Comment by soberich** [comment]
Hi, @jGauravGupta 
I have the same issue
Whethre 4.7 or earlier versions when try to install on Netbeans 9.0 screams 
```ome plugins require plugin Docker API to be installed.
The plugin Docker API is requested in implementation version 201705191307.  The following plugin is affected:       Docker 
Some plugins require plugin Docker UI to be installed.
The plugin Docker UI is requested in implementation version 201705191307.  The following plugin is affected:       Docker 
Some plugins require plugin J2EE Support Utilities to be installed.
The plugin J2EE Support Utilities is requested in implementation version 201609300101.  The following plugin is affected:       Infra
```
etc.
the list is long
This can be solved by by editinng Manifest.mf in each of those jars but there are too many of them.

I see you have a REQUIRMENT of netbeans 9.0 at the "microservices" project generation page.
So, how is this supposed to be installed?
Any workaround?
Thanks.

**Comment by jGauravGupta** [comment]
Sorry @soberich , it's a documentation mistake (it should be NetBeans IDE v8.2), I have corrected it.

**Comment by soberich** [comment]
@jGauravGupta 
Ok, anyway the tool is great,
is this one is also missing, or am I missing ?

![image](https://user-images.githubusercontent.com/25544967/36201896-a8d1558e-1192-11e8-8e0c-938e9f5d93db.png)

**Comment by jGauravGupta** [comment]
Issue closed as current release (v4.8.2) is available for NetBeans 8.2 Patch 2
